twilight-model/src/guild/invite/invite_type.rs

```diff
@@ -0,0 +1,66 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(from = "u8", into = "u8")]
+pub enum InviteType {
+    Guild,
+    GroupDM,
+    Friend,
+    Unknown(u8),
+}
+
+impl From<u8> for InviteType {
+    fn from(value: u8) -> Self {
+        match value {
+            0 => InviteType::Guild,
+            1 => InviteType::GroupDM,
+            2 => InviteType::Friend,
+            unknown => InviteType::Unknown(unknown),
+        }
+    }
+}
+
+impl From<InviteType> for u8 {
+    fn from(value: InviteType) -> Self {
+        match value {
+            InviteType::Guild => 0,
+            InviteType::GroupDM => 1,
+            InviteType::Friend => 2,
+            InviteType::Unknown(unknown) => unknown,
+        }
+    }
+}
+
+impl InviteType {
+    pub const fn name(&self) -> &str {
+        match self {
+            Self::Guild => "Guild",
+            Self::GroupDM => "Group DM",
```

Comment:
in other similar types we follow the name of the variant itself so it'd be `GroupDM`

Reply:
whats the reason you used `Group` instead of `GroupDm`